src/orchid/resources/posts/meetups/2023-11-08-meetup-3.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Meet Up: 21st November 2023"
+featuredImage: assets/media/2-banner.png
+tags:
+    - kotlin
+    - meetup
+    - kotlin multiplatform
+    - stable release
+---
+# 🎉 Exciting News Alert! 🚀
+
+Hey Kotlin Enthusiasts and Coding Connoisseurs,
+
+Get ready to witness a tech revolution like never before! 🌐✨ JetBrains is going to start with an online event, which will take place on November 21 and will uncover the state of Kotlin Multiplatform, and we want __YOU__ to be part of the grand reveal!
+
+📅 __Save the Date:__ 2023-11-21 </br>
+🕒 __Event Time:__ 18:30 - 20:30 </br>
+📍 __Location:__ Shift4 Office, A. Goštauto g. 40a-5th Floor, 01112
+
+### 🚀 What to Expect:
+
+🌐 How Kotlin Multiplatform works </br>
```

Comment:
I've just realized that this sentence sounds a bit odd. On the ti.to page I've rephrased into "Introduction into how Kotlin Multiplatform works"